### DISCUSSION TITLE
How do you use Material Design with MahApps

### DISCUSSION THREAD

**Discussion by Booth-Transport** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3040] [Q&A]
I was looking for some examples and what libraries and resources I need to use. I can't find any examples in the documentation. I'm mostly using Material Design except for when something is not provided. An example is Numericupdown

**Comment by nicolaihenriksen** [comment]
@Booth-Transport I think @Keboo has some MahApps examples in his [examples repo](https://github.com/Keboo/MaterialDesignInXaml.Examples/tree/master/MahApps). Perhaps there are also other hidden gems he can point you to.

**Comment by Keboo** [comment]
There is also a MahApps example in this repo as well: https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/tree/master/MahMaterialDragablzMashUp

**Comment by Booth-Transport** [comment]
I tried to set this up. But it throws the following error when I run the app and navigate to the page that has the numericupdown control. 
` Cannot find resource named 'MahApps.Styles.Button.Chromeless'. Resource names are case sensitive.`

There's no build error 

I've the latest versions of Materialdesign and MahApps from Nuget. I'm developing using .Net Framework 4.8. Below is my resource dictionary

```
 <ResourceDictionary.MergedDictionaries>
                <ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Controls.xaml" />
                <ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Fonts.xaml" />
                <ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.MahApps;component/Themes/MaterialDesignTheme.MahApps.Defaults.xaml" />
                <hc:ThemeResources/>
                <hc:Theme/>
                <materialDesign:BundledTheme BaseTheme="Light" PrimaryColor="DeepPurple" SecondaryColor="Lime" />
                <ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.Defaults.xaml" />
                <ResourceDictionary Source="pack://application:,,,/MaterialDesignExtensions;component/Themes/Generic.xaml" />
                <ResourceDictionary Source="pack://application:,,,/MaterialDesignExtensions;component/Themes/MaterialDesignLightTheme.xaml" />


                <!-- primary colors -->
                <ResourceDictionary>
                    <ResourceDictionary.MergedDictionaries>
                        <ResourceDictionary Source="pack://application:,,,/MaterialDesignColors;component/Themes/MaterialDesignColor.Blue.xaml" />
                    </ResourceDictionary.MergedDictionaries>
                    <SolidColorBrush x:Key="PrimaryHueLightBrush" Color="{StaticResource Primary100}" />
                    <SolidColorBrush x:Key="PrimaryHueLightForegroundBrush" Color="{StaticResource Primary100Foreground}" />
                    <SolidColorBrush x:Key="PrimaryHueMidBrush" Color="{StaticResource Primary500}" />
                    <SolidColorBrush x:Key="PrimaryHueMidForegroundBrush" Color="{StaticResource Primary500Foreground}" />
                    <SolidColorBrush x:Key="PrimaryHueDarkBrush" Color="{StaticResource Primary600}" />
                    <SolidColorBrush x:Key="PrimaryHueDarkForegroundBrush" Color="{StaticResource Primary600Foreground}" />
                    
                </ResourceDictionary>

                <!-- accent color -->
                <ResourceDictionary>
                    <ResourceDictionary.MergedDictionaries>
                        <ResourceDictionary Source="pack://application:,,,/MaterialDesignColors;component/Themes/MaterialDesignColor.Lime.xaml" />
                    </ResourceDictionary.MergedDictionaries>
                    <SolidColorBrush x:Key="SecondaryHueMidBrush" Color="{StaticResource Accent400}" />
                    <SolidColorBrush x:Key="SecondaryHueMidForegroundBrush" Color="{StaticResource Accent400Foreground}" />
                </ResourceDictionary>
            </ResourceDictionary.MergedDictionaries>
```

The control is coded like below

 <mah:NumericUpDown Style="{StaticResource MaterialDesignNumericUpDown}" />

**Reply by nicolaihenriksen** [reply]
@Booth-Transport I haven't tested this, and basically I am just guessing. Could it be that your `<materialDesign:BundledTheme ...>` should be a `<materialDesign:MahAppsBundledTheme ..>` instead?

**Reply by Booth-Transport** [reply]
Thanks for the response. This did not solve the problem. However I did some investigations myself and found out that this happens even if you remove the style from the numericupdown i.e. if you just have the following, it'll still error
`  <mah:NumericUpDown  />`

So I started removing resources one by one and saw that the error message goes away if I remove the following line
` <ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.MahApps;component/Themes/MaterialDesignTheme.MahApps.Defaults.xaml" />`

I got the package from Nuget that shows version 0.2.6.  Is that the version I should be using?
